src/games/GameList.tsx

```diff
@@ -31,7 +31,7 @@ const postFilters = [
 
 export const GameList = () => (
   <List filters={postFilters}>
-    <Datagrid>
+    <Datagrid isRowSelectable={() => false}>
```

Comment:
you should use `bulkActionButtons={false}` instead (it completely hides the checkboxes)

```suggestion
    <Datagrid bulkActionButtons={false}>
```